ncc/examples/monogram.c

```diff
@@ -0,0 +1,3536 @@
+#include <uvm/syscalls.h>
+#include <uvm/utils.h>
+
+
+// Monogram 12x7 pixel font
+
+// > # CREDITS
+// > 
+// > Monogram is a free and Creative Commons Zero pixel font,
+// > made by Vinícius Menézio (@vmenezio).
+// > 
+// > https://datagoblin.itch.io/monogram
+// > 
+// > 
+// > # SPECIAL THANKS
+// > 
+// > thanks to Ateş Göral (@atesgoral) for creating the bitmap font converter:
+// > https://codepen.io/atesgoral/details/RwGOvPZ
+// > 
+// > thanks to Éric Araujo (@merwok_) for the inital port of monogram to PICO-8:
+// > https://itch.io/post/2625522
+
+u64 font_monogram_number_of_characters = 390;
+u64 font_monogram_height = 12;
+u64 font_monogram_width = 7;
+
+u8 font_monogram_data[390][12];
+
+// Map from ord(ch) to index of ch's data in font_monogram_data.
+// This is for when you know the character that you want to draw but not
+// its index in the font_monogram_data array.
+u16 font_monogram_chars[8595];
+
+
+void
+init_monogram_font_data()
+{
+    memset(font_monogram_data, 0, sizeof(font_monogram_data));
+
+    // Empty slots initialized to -1.
+    memset(font_monogram_chars, 0xFF, sizeof(font_monogram_chars));
+
+    font_monogram_data[0][0] = 0;
+    // '0'
+    font_monogram_data[0][ 3] = 0b0001110;
+    font_monogram_data[0][ 4] = 0b0010001;
+    font_monogram_data[0][ 5] = 0b0011001;
+    font_monogram_data[0][ 6] = 0b0010101;
+    font_monogram_data[0][ 7] = 0b0010011;
+    font_monogram_data[0][ 8] = 0b0010001;
+    font_monogram_data[0][ 9] = 0b0001110;
+    font_monogram_chars[0x30] = 0;
```

Comment:
This makes me wish I had proper array initializer syntax in NCC so this could be more compact (but it doesn't matter for now).

Reply:
Yeah, my thoughts exactly.  Proper array initializer syntax would be lovely but its absence isn't a showstopper.